x/lsnative/slashing/types/tx.pb.go

```diff
@@ -106,8 +106,8 @@ func (m *MsgUnjailResponse) XXX_DiscardUnknown() {
 var xxx_messageInfo_MsgUnjailResponse proto.InternalMessageInfo
 
 func init() {
-	proto.RegisterType((*MsgUnjail)(nil), "cosmos.slashing.v1beta1.MsgUnjail")
-	proto.RegisterType((*MsgUnjailResponse)(nil), "cosmos.slashing.v1beta1.MsgUnjailResponse")
+	proto.RegisterType((*MsgUnjail)(nil), "lsnative.slashing.v1beta1.MsgUnjail")
```

Comment:
we are not switching it here?

Reply:
@puneet2019 good catch

Reply:
Sorry didn't get the question. We're replacing type registrations because of the name collision with built-in Cosmos types that are registered during `init()` of the corresponding packages.